deploy/crds/openshift_jenkins_v1alpha2_jenkins_cr.yaml

```diff
@@ -3,7 +3,7 @@ kind: Jenkins
 metadata:
   annotations:
     jenkins.io/openshift-mode: 'true'
```

Comment:
Can you remove this annotation ?

Reply:
I agree with Vibhav, please remove this line.